apps/nextjs-website/src/_contents/send/apiPath.ts

```diff
@@ -4,3 +4,13 @@ export const sendApiPath: Path = {
   name: 'API',
   path: '/send/api',
 };
+
+export const sendB2BApiPath: Path = {
+  name: 'B2B API',
```

Comment:
Avoid duplication of names with https://github.com/pagopa/developer-portal/blob/d59b4097d13e8cd92dfb666f9d58e5a18ce89c5d/apps/nextjs-website/src/_contents/send/api.ts#L12

Reply:
@marcobottaro it looks like you are not avoiding duplication with the file I linked in the comment.

The idea is to use the same variable you just created also [here](https://github.com/pagopa/developer-portal/blob/d59b4097d13e8cd92dfb666f9d58e5a18ce89c5d/apps/nextjs-website/src/_contents/send/api.ts#L12), so you are preventing any duplication errors. Moreover, the React component uses the names inside the `api.ts` file linked to select the proper OpenAPI specification, so if the names are different the page could show an error.